src/main/scala/vpio/IO.scala

```diff
@@ -66,15 +66,20 @@ object IO extends CommandApp(
 
           /* Assumes that OSM ORC is in LatLng */
           val latlngFeats: RDD[osm.OSMFeature] =
-            osm.toFeatures(ns.repartition(100), ws.repartition(10), rs)
+            osm.toFeatures(
+              VectorPipe.logToLog4j,
+              ns.repartition(100).map(_._2),
+              ws.repartition(10).map(_._2),
+              rs.map(_._2)
+            )
```

Comment:
This will change in future when the Planet History code supercedes/swallows the original Planet code, and Planet becomes a simple case of Planet History (where there's only one copy of everything).